### DISCUSSION TITLE
Help Needed: Output Folder Not Created After Running msnoise stack -r

### DISCUSSION THREAD

**Discussion by Thully1** [ROBelgium/MSNoise#366] [Q&A]
Hello everyone,

I'm currently processing some data using MSNoise and encountering an issue. I followed the documentation instructions and everything ran well up until the cross-correlation (CC) computations, which were marked as done. However, no output folder was created.

I ran msnoise stack -r, and the output included the following messages:
2024-09-19 20:26:26.761172 msnoise [pid 19228][INFO]: There are STACKS jobs for some days to recompute for CT.A9N:CT.A9U
2024-09-19 20:26:26.808047 msnoise [pid 19228][INFO]: There are STACKS jobs for some days to recompute for CT.A9G:CT.B4W
2024-09-19 20:26:26.839295 msnoise [pid 19228][INFO]: There are STACKS jobs for some days to recompute for CT.A9J:CT.A9P
Despite this, no folder was created, and I have tried manually creating the folder and updating the path in the configuration settings, but the issue persists.

Here's the information from msnoise info:
General:
 - the MSNoise module is installed in: C:\ProgramData\anaconda3\Lib\site-packages
 - db.ini is present in the current directory

Database information stored in the db.ini file:
 - database type: MySQL
 - hostname: 127.0.0.1
 - database: project1
 - username: ambient
 - password: ************
 - table prefix: %Tble%

Configuration:
 - D:\AMBIENT NOISE\Converted data\2023\CT exists
 - CROSS_CORRELATIONS does not exist and that is not normal (2426 CC jobs done)

Configuration values:
  
Jobs:
 CC:
  D : 2426
 STACK:
  D : 2426
 MWCS:
  none defined
 DTT:
  none defined

Has anyone encountered a similar issue or have suggestions on resolving it?
Are there any additional logs or configurations I should check?
Any help or insights would be greatly appreciated!

Thank you!

**Comment by ThomasLecocq** [comment]
Hi, how did you install msnoise? which version is it ?
